### DISCUSSION TITLE
Manage repos by creating folder

### DISCUSSION THREAD

**Discussion by hupo376787** [community/community#21795] [New to GitHub]
<p>Hi, I have stared many many repos. When I need to find a repo, I have to scroll page by page.</p>
<p>But Github now let me user search funtion, but it has limitations. What if the search keyword does not be contained in that repo? So the search result maybe none.</p>
<p>I suggest to use folder to manage repos. Folder can minimize the search range.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d2f2274b46ea577697c97c73bd6305fcaeaa972c.jpeg" data-download-href="/uploads/short-url/u674YvY1XohJBzeAZAKjJnZd13C.jpg?dl=1" title="无标题.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/14960168/181066921-d5041b00-da78-4216-b59e-987883a9da73.jpg" alt="无标题.jpg" data-base62-sha1="u674YvY1XohJBzeAZAKjJnZd13C" width="690" height="273" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/d2f2274b46ea577697c97c73bd6305fcaeaa972c_2_690x273.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/d2f2274b46ea577697c97c73bd6305fcaeaa972c_2_1035x409.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/d2f2274b46ea577697c97c73bd6305fcaeaa972c_2_1380x546.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d2f2274b46ea577697c97c73bd6305fcaeaa972c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">无标题.jpg</span><span class="informations">1444×573 59 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thx</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/hupo376787">@hupo376787</a>, thanks for being here and suggesting this! The best way to get feedback and feature ideas to GitHub is via this page: <a href="https://support.github.com/contact/feedback" rel="nofollow noopener">https://support.github.com/contact/feedback</a></p>
